proteuslib/unit_models/reverse_osmosis_0D.py

```diff
@@ -537,16 +547,15 @@ def recovery_mass_phase_comp_bounds(b, t, p, j):
             self.flowsheet().config.time,
             doc="Material properties of feed-side interface at outlet",
             default=tmp_dict)
-        # Permeate properties
-        self.properties_permeate = self.config.property_package.state_block_class(
+        self.permeate_side.properties_permeate = self.config.property_package.state_block_class(
             self.flowsheet().config.time,
             doc="Material properties of permeate",
             default=tmp_dict)
 
         # Add Ports
         self.add_inlet_port(name='inlet', block=self.feed_side)
         self.add_outlet_port(name='retentate', block=self.feed_side)
-        self.add_port(name='permeate', block=self.properties_permeate)
+        self.add_port(name='permeate', block=self.permeate_side.properties_permeate)
```

Comment:
Is there any difference here between using the `add_port` method as opposed to the `add_outlet_port` method?

Reply:
I think I realized the difference after the fact.

 `self.permeate_side.properties_permeate` is a state block, thus the use of `add_port()` (has no inlet/outlet). Meanwhile, `self.feed_side` is a control volume, hence the use of `add_outlet_port`.